core/src/apps/cardano/sign_tx.py

```diff
@@ -316,7 +289,40 @@ def _validate_metadata(metadata: Optional[bytes]) -> None:
         raise INVALID_METADATA
 
 
-def _serialize_tx(keychain: seed.Keychain, msg: CardanoSignTx) -> Tuple[bytes, bytes]:
+async def _send_signed_tx(ctx, keychain: seed.Keychain, msg: CardanoSignTx) -> None:
```

Comment:
I feel like this function is doing a bit too much work and the name is not accurate then. A proper name would be something like `_serialize_and_send_chunked_tx`. I would consider passing in the `cborized_tx` and `tx_hash`.

Reply:
agree, done